test/extended/testdata/builds/test-context-build.json

```diff
@@ -42,7 +42,7 @@
           "git": {
             "uri":"https://github.com/sclorg/s2i-ruby-container"
           },
-          "contextDir": "3.0/test/puma-test-app"
+          "contextDir": "3.3/test/puma-test-app"
```

Comment:
You should also change the "from" image to reference the ImageStream being created in the test. See [doc](https://docs.openshift.com/container-platform/4.15/cicd/builds/build-strategies.html#builds-strategy-s2i-build_build-strategies-docker)